samples/04_gis_analysts_data_scientists/information_extraction_from_cheshire_fire_incident_reports_using_mistral_language_model.ipynb

```diff
@@ -0,0 +1,1208 @@
+{
```

Comment:
<p>Now that we have the trained model, let's look at how the model performs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>